modules/formulize/class/data.php

```diff
@@ -607,7 +607,7 @@ function findAllEntriesWithValue($element_id, $value, $scope_uids=array(), $scop
 		global $xoopsDB;
     $form_handler = xoops_getmodulehandler('forms', 'formulize');
     $formObject = $form_handler->get($this->fid);
-		$queryValue = "\"" . formulize_db_escape($value) . "\"";
+		$queryValue = "'" . formulize_db_escape($value) . "'";
```

Comment:
We need to test things going in and out of the db with this change. This is fundamental. And what happens to single and double quotes now that we're single quoting... ack. Prediction, they're all special characters. But when they're not special characters, this will still work right??? Need to verify now.